metadata-io/src/test/java/com/linkedin/metadata/graph/GraphServiceTestBase.java

```diff
@@ -938,17 +939,17 @@ public void testFindRelatedEntitiesNullDestinationType() throws Exception {
     assertNotNull(nullUrn);
     RelatedEntity nullRelatedEntity = new RelatedEntity(downstreamOf, nullUrn.toString());
 
-    doTestFindRelatedEntitiesEntityType(anyType, "null", downstreamOf, outgoingRelationships, service);
+    doTestFindRelatedEntitiesEntityType(anyType, ImmutableList.of("null"), downstreamOf, outgoingRelationships, service);
```

Comment:
Same?

Reply:
yep! in the test there is actually an entity type of "null"